source/pages/Settings/CustomRPC.tsx

```diff
@@ -76,6 +73,14 @@ const CustomRPCView = () => {
     explorer: (state && state.selected && state.selected.explorer) ?? '',
   };
 
+  const canDisableInputWhenEdit = state ? state.isDefault : false;
+
+  const canDisableDecimalsInput = Boolean(
```

Comment:
I think we can change this to something like `isDecimalsInputDisabled` sound more like the use case here. IMHO `can` looks like the user can do some action or not based on the value